server/src/service.rs

```diff
@@ -114,16 +114,70 @@ impl Service {
     }
 
     #[zbus(out_args("collection"))]
-    pub async fn read_alias(&self, _name: &str) -> Result<ObjectPath, ServiceError> {
-        todo!()
+    pub async fn read_alias(
+        &self,
+        name: &str,
+        #[zbus(object_server)] object_server: &zbus::ObjectServer,
+    ) -> Result<OwnedObjectPath, ServiceError> {
+        let collections = self.collections.lock().await;
+
+        for collection in collections.iter() {
+            let collection_ifce_ref = object_server.interface::<_, Collection>(collection).await?;
+            let collection = collection_ifce_ref.get_mut().await;
+
+            if name == "login" {
+                if collection.label().await == name {
+                    drop(collections);
+                    return Ok(collection.path().clone());
+                }
+            } else if collection.alias().await == name {
+                drop(collections);
+                return Ok(collection.path().clone());
+            }
+        }
+
```

Comment:
Log which collection wasn't found

Reply:
Done, thanks!

Reply:
This was removed?

Reply:
> Log which collection wasn't found

I think this comment was related to service.rs:161

Reply:
Do we really need logging in ReadAlias call?

Reply:
Yes.

Reply:
ok, I'm on it.

Reply:
Done.